srcpkgs/kitty/template

```diff
@@ -1,19 +1,25 @@
 # Template file for 'kitty'
 pkgname=kitty
-version=0.10.1
+version=0.11.1
 revision=1
 python_version=3
-hostmakedepends="ncurses pkg-config python3-devel wayland-devel wayland-protocols"
+hostmakedepends="ncurses pkg-config"
 makedepends="gettext-devel glfw-devel harfbuzz-devel libxkbcommon-devel
  python3-devel wayland-devel wayland-protocols"
 depends="kitty-terminfo"
 conf_files="/etc/kitty/kitty.conf"
 short_desc="Modern, hackable, featureful, OpenGL based terminal emulator"
 maintainer="Julio Galvan <juliogalvan@protonmail.com>"
 license="GPL-3.0-or-later"
-homepage="https://github.com/kovidgoyal/kitty"
-distfiles="https://github.com/kovidgoyal/kitty/archive/v${version}.tar.gz"
-checksum=ef22208497a76e2f88ebe56c176e4608f049b056252cf1bf122c9c1ec711cfa6
+homepage="https://sw.kovidgoyal.net/kitty/"
+distfiles="https://github.com/kovidgoyal/kitty/releases/download/v${version}/kitty-${version}.tar.xz"
+checksum=3baf72ee4de8818bbc205fd631f716b60eabc8bd215ec7e834e00a01db7912c4
+
+if [ -n "$CROSS_BUILD" ]; then
+	hostmakedepends+=" python3-devel wayland-devel wayland-protocols
+	 libxkbcommon xkeyboard-config xkeyboard-config libxkbfile
```

Comment:
`xkeyboard-config xkeyboard-config`

twice the same?

Reply:
this only needs ` python3-devel wayland-devel wayland-protocols kitty`. xkeyboard-config issue is on voids side and other stuff is absolutely unnecessary.

Reply:
#477 is supposed to fix this

Reply:
ok, rebuilding with only ` python3-devel wayland-devel wayland-protocols`